logdna/common_test.go

```diff
@@ -110,7 +114,7 @@ func fmtProviderBlock(args ...string) string {
 	return fmt.Sprintf(tmplPc, pcCfg)
 }
 
-func fmtResourceBlock(objTyp, rsName string, rsArgs map[string]string, chArgs map[string]map[string]string) string {
+func fmtResourceBlock(objTyp, rsName string, rsArgs map[string]string, chArgs map[string]map[string]string, dependencies []string) string {
```

Comment:
this got a little dicey with all the areas of code it impacts - we should consider refactoring this, not necessarily here in this PR

Reply:
@mdeltito yes, it makes sense, @darinspivey also mentioned this point, I'll work on the refactoring in a separate story.